fedimint-server/src/net/api.rs

```diff
@@ -296,6 +310,73 @@ impl ConsensusApi {
             .await
             .map_err(|_| ApiError::server_error("Unable send event".to_string()))
     }
+
+    pub async fn build_full_server_status(&self) -> ApiResult<FullStatus> {
+        let our_epoch = self.get_epoch_count().await;
+        // Clone to avoid holding the lock while we calculate things
+        let rejoin_at_epoch = self.rejoin_at_epoch.lock().await.clone();
+        let last_epoch_by_peer = rejoin_at_epoch.into_iter().fold(
+            HashMap::<PeerId, u64>::new(),
+            |mut acc, (epoch, peers)| {
+                let epoch = epoch + 1;
```

Comment:
I'm not sure why `+ 1`.

Reply:
AFAIK, the reason is the same why there is a `+ 1` here:
https://github.com/fedimint/fedimint/blob/60595f294c854a24fad47243ec4d42ad67603a59/fedimint-server/src/net/api.rs#L268

Reply:
Oh. So it's `let epoch_count = epoch + 1;`. Maybe even `epoch` should be `last_epoch` or something. Or maybe not.